benchmarks/operator_benchmark/pt/qpool_test.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+import torch.quantization as tq
+
+import operator_benchmark as op_bench
+
+# 2D pooling will have input matrix of rank 3 or 4
+qmaxpool2d_configs = op_bench.config_list(
+    attrs=(
+       #  C    H    W   k       s       p       d
+       (  1,   3,   3, (3, 3), (1, 1), (0, 0), (1, 1)),  # dummy        # noqa
+       (  3,  64,  64, (3, 3), (1, 1), (0, 0), (1, 1)),  # dummy        # noqa
+       (  3,  64,  64, (3, 3), (2, 2), (1, 1), (2, 2)),  # dummy        # noqa
+       # VGG16 pools with original input shape: (-1, 3, 224, 224)
+       ( 64, 224, 224, (2, 2), (2, 2), (0, 0), (1, 1)),  # MaxPool2d-4  # noqa
+       (128, 112, 112, (2, 2), (2, 2), (0, 0), (1, 1)),  # MaxPool2d-9  # noqa
+       (256,  56,  56, (2, 2), (2, 2), (0, 0), (1, 1)),  # MaxPool2d-16 # noqa
+       (512,  28,  28, (2, 2), (2, 2), (0, 0), (1, 1)),  # MaxPool2d-23 # noqa
+       (512,  14,  14, (2, 2), (2, 2), (0, 0), (1, 1)),  # MaxPool2d-30 # noqa
+    ),
+    attr_names=('C', 'H', 'W',  # Input layout
+                'k', 's', 'p', 'd',  # Pooling parameters
+                            ),
+    cross_product_configs={
+        'N': range(5),  # if N==0, use rank=3
+        'ceil': (False, True),
+        'contig': (False, True),
+        'dtype': (torch.qint32, torch.qint8, torch.quint8),
+    },
+    tags=('long',)
```

Comment:
Consider marking it as `short` which is the default mode to avoid tag_filter.

Reply:
Oh, I thought long and short run on different machines. My long tests are REALLY long -- there are like 7000 combinations for the benchmark parrameters in some of them :) Should I still mark it as short?

Reply:
long and short will run on the same machine. 7000 combinations seem too much. Right now, each test(a single test of an operator) runs around 10 seconds. With 7000 combinations in one operator, it will take around 7000 * 5 second / 3600 = 9 hours. In our current testing, we are aiming to run all quantized ops within 8 hours. Could you take a look at this and those other tests try to fit that 8 hours time frame? You may want to split the input shapes into short and long. In that case, quick perf verification is possible.